### DISCUSSION TITLE
Thank you for TrayWeather

### DISCUSSION THREAD

**Discussion by enervated** [FelixdelasPozas/TrayWeather#85] [General]
This application is great and continues to get better. You make my life a little better, and I appreciate it.

Thank you!

**Comment by FelixdelasPozas** [comment]
Thanks @enervated ! Much appreciated. If it keeps getting better is because you all are still reporting bugs and asking for new things!

Kind regards, 
Félix.

**Comment by enervated** [comment]
Is there a way I can donate?

**Comment by FelixdelasPozas** [comment]
Thanks, it's not necessary but I really appreciate it! Some people told me to get a "buy me a coffee" page for donations but the fact that you use the program and like it is reward enough. 

Kind regards, 
Félix.

**Comment by KenLowe65** [comment]
Works great in Canada, on Windows 10.
I've made a small donation to a local animal rescue group to share the positive vibes.
Thank you!

**Reply by FelixdelasPozas** [reply]
Thank you! Much appreciated!!

**Comment by enervated** [comment]
For any users coming across this thread, I just wanted to update it to mention that a Ko-fi donation method has been made available. You can access it from the application's About box, or use the link below:

https://ko-fi.com/felixdelaspozas

**Reply by FelixdelasPozas** [reply]
Thank you very much!
